### PR TITLE
Update es.m3u

### PR DESCRIPTION
**Pull request by trancer1994**
Spain (including Basque Terrotories and Catelonia (if you want to split these up in the future you can do so))

### CODE REVIEW

File: channels/es.m3u [hunk 1]
```diff
@@ -601,3 +601,80 @@ https://cloud.streamingconnect.tv:455/radiotvzafra/radiotvzafra.m3u8
...
+https://directes-tv-int.ccma.cat/int/ngrp:c33_web/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/sEN6VcY.png" group-title="",Canal Sur Andalucía
+https://originlive2.shooowit.net/rtvalive/smil:channelDVR.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/1RVS7Wh.png" group-title="",Canalcosta TV
+https://5d8d85cf2c308.streamlock.net:1936/CanalcostaTV/WXP6YT/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/QCKNWMJ.jpg" group-title="",TV3 Catalunya
+https://directes-tv-int.ccma.cat/int/ngrp:tv3_web/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV3CAT
+https://directes-tv-int.ccma.cat/int/ngrp:tvi_web/playlist_DVR.m3u8
+Norway:
```
Comment (freearhey) [inline channels/es.m3u:680]: Invalid line.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please remove extra line.

**Comment by freearhey** [comment]
We divide channels into files based on the list of countries from [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) and that is why only Spain is on the list.

**Comment by freearhey** [comment]
Closed due to lack of any activity.

**Comment by trancer1994** [comment]
It says "This pull request is closed, but the trancer1994:patch-34 branch has unmerged commits". What do I do now?

**Comment by freearhey** [comment]
@trancer1994 You can leave it as it is. It's not an error, just a notification.

**Pull request closed without merging**